src/test/resources/json/sample.json

```diff
@@ -32,12 +32,29 @@
                     },
                     {
                         "result":{
+                            "duration":9520000,
+                            "status":"passed"
+                        },
+                        "name":"My credit card is described as follow:",
+                        "keyword":"And ",
+                        "line":9,
+                        "match":{
+                            "location":"ATMScenario.My_credit_card_is_described_as_follow"
+                        },
+                        "doc_string": {
+                            "content_type": "",
```

Comment:
The content_type is an user specified value, for instance:
"""json
{"field": "value"}
"""

Here I just left it out for simplicity :)